src/ipahealthcheck/core/core.py

```diff
@@ -243,6 +246,15 @@ def run_healthcheck(self):
         options = parse_options(self.parser)
         self.options = options
 
+        if options.version:
+            for registry in self.entry_points:
+                name = registry.split('.')[0]
+                print(
+                    '%s: %s' %
+                    (name, pkg_resources.get_distribution(name).version)
```

Comment:
This makes unconditional dependency on `pkihealthcheck` and fails if it's not installed:
```
E           ipapython.ipautil.CalledProcessError: CalledProcessError(Command ['ipa-healthcheck', '--version'] returned non-zero exit status 1: 'Traceback (most recent call last):\n  File "/usr/src/RPM/BUILD/freeipa-healthcheck/.tox/py3/bin/ipa-healthcheck", line 8, in <module>\n    sys.exit(main())\n  File "/usr/src/RPM/BUILD/freeipa-healthcheck/.tox/py3/lib/python3/site-packages/ipahealthcheck/core/main.py", line 46, in main\n    sys.exit(ipachecks.run_healthcheck())\n  File "/usr/src/RPM/BUILD/freeipa-healthcheck/.tox/py3/lib/python3/site-packages/ipahealthcheck/core/core.py", line 254, in run_healthcheck\n    (name, pkg_resources.get_distribution(name).version)\n  File "/usr/src/RPM/BUILD/freeipa-healthcheck/.tox/py3/lib/python3/site-packages/pkg_resources/__init__.py", line 466, in get_distribution\n    dist = get_provider(dist)\n  File "/usr/src/RPM/BUILD/freeipa-healthcheck/.tox/py3/lib/python3/site-packages/pkg_resources/__init__.py", line 342, in get_provider\n    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]\n  File "/usr/src/RPM/BUILD/freeipa-healthcheck/.tox/py3/lib/python3/site-packages/pkg_resources/__init__.py", line 886, in require\n    needed = self.resolve(parse_requirements(requirements))\n  File "/usr/src/RPM/BUILD/freeipa-healthcheck/.tox/py3/lib/python3/site-packages/pkg_resources/__init__.py", line 772, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound: The \'pkihealthcheck\' distribution was not found and is required by the application\n')
```

Reply:
sorry for formatting.

Reply:
I'm not sure when this may happen but I catch and ignore the exception now.

Reply:
This happens when I run freeipa-healthcheck self-tests and there is no dependency on pki-healthcheck right now.

Reply:
`core` package shouldn't have deps on plugins.